websockets.cabal

```diff
@@ -81,7 +82,9 @@ Library
     random            >= 1.0    && < 1.2,
     SHA               >= 1.5    && < 1.7,
     text              >= 0.10   && < 1.3,
-    entropy           >= 0.2.1  && < 0.4
+    entropy           >= 0.2.1  && < 0.4,
+    streaming-commons >= 0.1.15,
+    safe-exceptions   >= 0.1.4.0
```

Comment:
I don't mind relying on `safe-exceptions`, but I am interested in why this was necessary?